### PR TITLE
Update remove-former-employee-step-2.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the version of outlook as we cannot backup data to PST from Web app.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5409

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/admin/add-users/remove-former-employee-step-2.md [hunk 1]
```diff
@@ -40,4 +40,4 @@ Once you've blocked a user from being able to log into your organization you can
 
     **OR**
 
-2. Add the former employee's email address to your version of Outlook web app, and then export the data to a .pst file. You can import the data to another email account as needed. Check out [Step 6 - Give another employee access to OneDrive and Outlook data](remove-former-employee-step-6.md).
+2. Add the former employee's email address to your version of Outlook Desktop app, and then export the data to a .pst file. You can import the data to another email account as needed. Check out [Step 6 - Give another employee access to OneDrive and Outlook data](remove-former-employee-step-6.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee-step-2.md]:
```suggestion
2. Add the former employee's email address to your version of Outlook on Desktop, and then export the data to a .pst file. You can import the data to another email account as needed. Check out [Step 6 - Give another employee access to OneDrive and Outlook data](remove-former-employee-step-6.md).
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**